orchagent/port/portcap.cpp

```diff
@@ -0,0 +1,78 @@
+// includes -----------------------------------------------------------------------------------------------------------
```

Comment:
Can we please rename to `portcapblty.cpp` or similar? `portcap `is slightly misleading.